src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,137 @@
+p,
+h2 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px 0px 0px 0px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), border-radius should be consistent. It looks like you've set border-radius only for the top-left corner. Ensure that the border-radius is applied consistently or as required by the design.